distributions/aws/test/e2e/fixtures/cluster.py

```diff
@@ -87,3 +87,23 @@ def on_delete():
     return configure_resource_fixture(
         metadata, request, cluster_name, "cluster_name", on_create, on_delete
     )
+
+
+def create_iam_service_account(
```

Comment:
Looks like this function is already defined on line 48, maybe the old one can be removed

Reply:
oh yea, done.